core/lib/globals_api.h

```diff
@@ -377,9 +377,9 @@ typedef struct {
  */
 typedef struct {
 #    ifdef X64
-    uint black_box_uint[28];
+    uint black_box_uint[30];
```

Comment:
2 more?  Because of padding?  Please try to move the field to eliminate the extra uint of space.